docs/index.md

```diff
@@ -446,6 +446,10 @@ f\filter(f\equals('Miles Davis'), $names); // ['Miles Davis']
 // I will preserve string indexes:
 $numbers = ['hundred' => 100, 'three' => 3, 'fiftytwo' => 52];
 f\filter(f\gt(50), $numbers); // ['hundred' => 100, 'fiftytwo' => 52]
+
+// I will remove falsy values:
+$values = [0, null, false, '', '0'];
+f\filter(f\id, $values); // []
```

Comment:
Hmm, although this is a neat solution, I'm not sure whether I want to present this as the principal way of filtering falsy values, since you can use `f\truthy` for exactly that purpose. 
I would argue `f\truthy` is a lot more self-documenting in this case than `f\id`, since the actual _mechanism_ that's filtering the values is still PHP's type coercion and the under-the-surface mapping of `f\filter` to `array_filter`. So this requires quite some insight into the workings of the function, whereas `f\truthy` describes exactly which values should end up in the sieve and which should pass through...

Reply:
🤦‍♂️